exonum/src/runtime/mod.rs

```diff
@@ -386,6 +386,24 @@ pub trait Runtime: Send + fmt::Debug + 'static {
         parameters: Vec<u8>,
     ) -> Result<(), ExecutionError>;
 
+    /// Resumes previously stopped service instance with the given specification and arguments.
+    /// For example, by this arguments service instance can update configuration.
```

Comment:
Grammar: "As an example, arguments can be used to update the service configuration" reads better, IMO.